src/ArticleList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import axios from 'axios'
+import ArticleCard from './ArticleCard'
+import './ArticleList.css'; 
+
+const ArticleList = () => {
+    const [articles, setArticles] = useState([]);
+    useEffect(() => {
```

Comment:
I would strongly recommend you extract this api call to a separate file - it is best practice and will make your components much cleaner if you do this for all other api calls.